js/src/View/TodoListView.js

```diff
@@ -0,0 +1,116 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var TodoListModel = require('../Model/TodoListModel.js');
+var TodoModel = require('../Model/TodoModel.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoCollection = require('../Collection/TodoCollection.js');
+var TodoView = require('./TodoView.js')
+
+
+var todoListViewTemplate = require('../templates/todolist.hbs');
+
+
+var TodoListView = Backbone.View.extend({
+    //why does $ syntax not work here?
+
+    tagName: 'div',
+    className: 'todo-list-container',
+
+    events: {
+        'click .deleteList': 'deleteList',
+        'keypress .todo': 'addTodoKeyPress',
+        'click .addTodo': 'addTodoClick',
+        'keypress .listName': 'updateList',
+    },
+
+    initialize: function(){
+        // create todoList model
+        this.renderTodoList();
+
+        // create todo collection
+        this.todos = new TodoCollection([], {id: this.model.id});
+        this.todos.fetch({
+            success: _.bind(function(model){
+                console.log('Success: Todos loaded.');
+                this.renderTodos();
+
+            }, this),
+            error: function(){
+                console.log('Error: Todos failed to load.');
+            }
+        })
+
+    },
+
+    renderTodoList: function(){
+        var todoList = this.model.toJSON();
+        this.el.innerHTML = todoListViewTemplate({listName: todoList.name});
```

Comment:
Totally optional, but I just wanted to point out that `todoList` here is an object, so if you wanted to, you could just pass it directly into the `todoListViewTemplate`. It gives the template a little more information than it needs, and the template would need to be update to use `name` instead of `listName`. Totally up to you. I like either approach, but I just wanted to point that you could do that if you wanted to.